Source/NSMenuItemCell.m

```diff
@@ -736,7 +784,34 @@ - (void) drawKeyEquivalentWithFrame: (NSRect)cellFrame
    */
   else if (![[_menuView menu] _ownedByPopUp] || (_imageToDisplay == nil))
     {    
-      [self _drawText: [self _keyEquivalentString] inFrame: cellFrame];
+      if (_keyEquivalentFont != nil)
+        {
+          NSDictionary       *attrs;
+          NSArray            *attrObjects, *attrKeys;
+          NSAttributedString *aString;
+          NSColor            *aColor;
+
+          if (_cell.is_disabled)
+            aColor = [NSColor disabledControlTextColor];
+          else
+            aColor = [NSColor controlTextColor];
```

Comment:
Here we should use [self textColor]. (See NSCell)

Reply:
Thanks. Fixed.